driver-core/src/test/java/com/datastax/driver/core/exceptions/ExceptionsScassandraTest.java

```diff
@@ -0,0 +1,144 @@
+/*
+ *      Copyright (C) 2012-2015 DataStax Inc.
+ *
+ *   Licensed under the Apache License, Version 2.0 (the "License");
+ *   you may not use this file except in compliance with the License.
+ *   You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *   Unless required by applicable law or agreed to in writing, software
+ *   distributed under the License is distributed on an "AS IS" BASIS,
+ *   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *   See the License for the specific language governing permissions and
+ *   limitations under the License.
+ */
+package com.datastax.driver.core.exceptions;
+
+import java.util.List;
+import java.util.Map;
+
+import com.google.common.collect.ImmutableList;
+import com.google.common.collect.ImmutableMap;
+import org.scassandra.http.client.PrimingRequest;
+import org.testng.annotations.*;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.fail;
+import static org.scassandra.http.client.PrimingRequest.Result.read_request_timeout;
+import static org.scassandra.http.client.PrimingRequest.Result.unavailable;
+import static org.scassandra.http.client.PrimingRequest.Result.write_request_timeout;
+
+import com.datastax.driver.core.*;
+import com.datastax.driver.core.policies.FallthroughRetryPolicy;
+
+import static com.datastax.driver.core.ConsistencyLevel.ONE;
+
+public class ExceptionsScassandraTest {
+
+    protected SCassandraCluster scassandras;
+    protected Cluster cluster;
+    protected Metrics.Errors errors;
+    protected Host host1;
+    protected Session session;
+
+    @BeforeClass(groups = "short")
+    public void beforeClass() {
+        scassandras = new SCassandraCluster(CCMBridge.IP_PREFIX, 1);
+    }
+
+    @BeforeMethod(groups = "short")
+    public void beforeMethod() {
+        cluster = Cluster.builder()
+            .addContactPoint(CCMBridge.ipOfNode(1))
+            .withRetryPolicy(FallthroughRetryPolicy.INSTANCE)
+            .build();
+        session = cluster.connect();
+        host1 = TestUtils.findHost(cluster, 1);
+        errors = cluster.getMetrics().getErrorMetrics();
+        scassandras.clearAllPrimes();
+        scassandras.clearAllRecordedActivity();
+    }
+
+    @Test(groups = "short")
+    public void should_throw_proper_unavailable_exception() {
```

Comment:
Mmm, actually the CCM-backed tests are still there in `ExceptionsTest`. Shouldn't we remove them for the classes that are covered here?